### ISSUE TITLE
Token Para 0x32879B5C0fDbc2A81597F019717b412B9d755A09 Doesn't show balance

### ISSUE THREAD

**Issue by Rachel-Eichenberger** [Uniswap/interface#3319]
Bug Description
The token PARA 0x32879B5C0fDbc2A81597F019717b412B9d755A09 is not pulling balance, circle just spins

Steps to Reproduce

Go to https://app.uniswap.org/#/swap
Click on token and select PARA 0x32879B5C0fDbc2A81597F019717b412B9d755A09
Balance will just spin

Expected Behavior
Load token balance in app.

Additional Context
@NoahZinsmeister 
perhaps this is a gas issue like the cat token recently had https://github.com/Uniswap/interface/issues/2936
I was curious why the gas in a call, but perhaps that's an etherjs thing

![image](https://user-images.githubusercontent.com/60412342/154204296-84c6ab15-d068-463d-9b51-bfdd782d1113.png)
![image](https://user-images.githubusercontent.com/60412342/154204300-744dcf74-11ce-4361-a51d-eacc09bd8650.png)

This token does show a balance in a older version of uniswap here
https://bafybeihbmaae4y42o5xojkmllmdcnph5cajdhxkvauqu2nceewtq5aokcu.ipfs.cf-ipfs.com/#/swap

**Comment by Keu25** [comment]
I am having this issue also.  Is there a fix in the works?  I can load ParaToken in the 1inch Platform and it loads.

Thank you

![Token wont load](https://user-images.githubusercontent.com/99854325/154395929-fa02fddd-ca82-44e0-ad01-c2b2b6c7dd8f.jpg)
![No tokens in Swap feature](https://user-images.githubusercontent.com/99854325/154396096-d2c5178f-cb3b-445d-b4c5-1647eb71e534.jpg)
![1 inch tokens work](https://user-images.githubusercontent.com/99854325/154396328-83eb9cf1-7dd9-436c-aae3-8dd5292a946a.jpg)

**Comment by willpote** [comment]
Para is working now, closing

![Screen Shot 2022-02-22 at 11 28 47 AM](https://user-images.githubusercontent.com/81638931/155175652-500c9da4-7457-47c9-a345-91f33e7fb447.png)

**Comment by Rachel-Eichenberger** [comment]
@willpote So this wasn't specifically about the trade route, but the balance of the tokens not showing, as in your screenshot, it doesn't show 0 tokens (or para balance) of users wallet

**Comment by willpote** [comment]
Ah ok, sorry, reopening.

**Comment by Keu25** [comment]
Correct.  The ParaToken does load, but does not show an amount of tokens or the value in ETH.   I am not sure where Uniswap pulls data from in showing these amounts.

Trading is a process as well.  Usually have to turn the api router off.  One must enter in token amounts manually.

If one uses se Sushi swap or 1inch, everything functions as Uniswap used to functuon.

Kind of a head scratcher lol.

Thanks for your continued efforts.


________________________________
